resources/definitions/flsun_sr.def.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": 2,
+    "name": "FlSun SuperRacer",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Thushan Fernando",
+        "manufacturer": "Flsun",
+        "platform": "flsun_sr.3mf",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+		"has_machine_quality": true,
```

Comment:
```suggestion
		"has_machine_quality": false,
```
The machine states that it has machine qualities but there are no quality profiles provided. If you don't keep it as false, you will end up with unsupported profiles. 
![image](https://user-images.githubusercontent.com/19388042/125798084-dbf8b85f-b927-46f2-abb1-6eaf448af02a.png)

If you set it to false, then the global qualities will be able to be used instead.
![image](https://user-images.githubusercontent.com/19388042/125799110-d673145e-0214-4408-a955-8b60fe7e4b42.png)

Reply:
Good catch, resolved as requested.

Will add quality profiles after consulting with FLSun. Good catch :)